package.json

```diff
@@ -30,14 +30,16 @@
   "browser": "dist/geofire.min.js",
   "module": "dist/index.esm.js",
   "typings": "dist/index.d.ts",
+  "utils": "dist/geofire-utils.js",
   "files": [
     "dist/**",
     "LICENSE",
     "README.md",
     "package.json"
   ],
   "dependencies": {
-    "@firebase/database-types": "0.x.x"
+    "@firebase/database-types": "0.x.x",
+    "@rollup/plugin-commonjs": "^15.0.0"
```

Comment:
If you need more `rollup` deps move them to `devDependencies` so they don't actually ship.

Reply:
My mistake: the plugin was already in `devDependencies`, so I rolled back to that version. No sense in updating all dependencies for just this new release target.